frame/executive/src/lib.rs

```diff
@@ -897,17 +897,32 @@ mod tests {
 		t.into()
 	}
 
+	fn new_test_ext_v0(balance_factor: Balance) -> sp_io::TestExternalities {
```

Comment:
Where is this used?

Reply:
just bellow in 'blocks_works' test (do no harm testing it).

Reply:
Sorry, I'm blind.